opensrp-chw/src/crvs/assets/json.form/child_enrollment.json

```diff
@@ -328,6 +357,397 @@
+        ],
+        "keys": [
+          "Césarienne",
+          "Forceps",
+          "Accouchement normal",
+          "Ventouse",
+          "Inconnu"
+        ],
+        "openmrs_choice_ids": {
+          "C-Section": "c_section",
+          "Forceps": "forceps",
+          "Normal delivery": "normal_delivery",
+          "Vacuum": "vacuum",
+          "Unknown": "unknown"
+        }
+      },
+      {
+        "key": "type_of_pregnancy",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "typeOfPregnancy",
+        "openmrs_data_type": "select one",
+        "type": "spinner",
+        "hint": "{{child_enrollment.step1.type_of_pregnancy.hint}}",
+        "values": [
+          "{{child_enrollment.step1.type_of_pregnancy.value[0]}}",
+          "{{child_enrollment.step1.type_of_pregnancy.value[1]}}",
+          "{{child_enrollment.step1.type_of_pregnancy.value[2]}}",
+          "{{child_enrollment.step1.type_of_pregnancy.value[3]}}",
+          "{{child_enrollment.step1.type_of_pregnancy.value[4]}}"
+        ],
+        "openmrs_choice_ids": {
+          "Singleton": "singleton",
+          "Twin": "twin",
+          "Triplet": "triplet",
+          "Other": "other",
+          "Unknown": "unknown"
+        },
+        "v_required": {
+          "value": "true",
+          "err": "{{child_enrollment.step1.type_of_pregnancy.error}}"
+        }
+      },
+      {
+        "key": "birth_weight",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "",
+        "label": "Baby Weight",
+        "hint": "{{child_enrollment.step1.birth_weight.hint}}",
+        "type": "edit_text",
+        "v_numeric": {
+          "value": "true",
+          "err": "{{child_enrollment.step1.birth_weight.v_numeric.error}}"
+        },
+        "v_min": {
+          "value": "500",
+          "err": "{{child_enrollment.step1.birth_weight.v_min.error}}"
+        },
+        "v_max": {
+          "value": "6000",
+          "err": "{{child_enrollment.step1.birth_weight.v_max.error}}"
+        }
+      },
+      {
+        "key": "mother_name",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "motherName",
+        "openmrs_data_type": "text",
+        "type": "edit_text",
+        "hint": "{{child_enrollment.step1.mother_name.hint}}",
+        "look_up": "true",
+        "edit_type": "name",
+        "v_required": {
+          "value": "true",
+          "err": "{{child_enrollment.step1.mother_name.error}}"
+        }
+      },
+      {
+        "key": "mother_id",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "motherId",
+        "type": "edit_text",
+        "hint": "{{child_enrollment.step1.mother_id.hint}}",
+        "edit_type": "name"
+      },
+      {
+        "key": "mother_dob_entered",
```

Comment:
There's no relevance attached to either show the date picker or  dob unknown checkbox